hash_practice/exercises.py

```diff
@@ -2,18 +2,46 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)
     """

```

Comment:
👍 , Time/space are correct if the words are limited in length (like English words) Otherwise it would be O(n * m log m) where m is the length of each word.